### ISSUE TITLE
FE centering map on Germany

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#201]
## Description
See Epic: #193 
If no region is search / selected center on germany as a whole

## Acceptance criteria
* prevent panning away from germany 
* Zooming out centers on germany once certain level is reached

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
max zoom plus boundaries = "centered"
